src/mock-categories.json

```diff
@@ -28,5 +28,10 @@
       "tid":"273",
       "name":"What Are You Solving For?",
       "description":"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+   },
+   {
+     "tid": "1118",
+     "name":"What We're Reading... And Our POV On It!",
+     "description":"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
    }
```

Comment:
As you'll need the `tid` for podcasts, after all, just go ahead and download the latest version of this export here: 
https://admin.rackspace.com/api/thought-leadership-categories?_format=json